flake.nix

```diff
@@ -45,7 +45,7 @@
               modules ++ [
                 {
                   system.nixos.versionSuffix =
-                    ".${final.substring 0 8 (self.lastModifiedDate or self.lastModified)}.${self.shortRev or "dirty"}";
+                    ".${final.substring 0 8 (self.lastModifiedDate or self.lastModified or "dirty")}.${self.shortRev or "dirty"}";
```

Comment:
Since it's a date, maybe `19691231` (the `mdate` on files in the store)?

Reply:
Not sure though. I needed a moment to mentally parse this, so at least to me it seems as `dirty` is more obvious :)

Reply:
I understand. Switched the fallback to `19700101` as this is more obvious IMHO :) 
Btw I think it would make sense to push this to 20.09 at least.